corehq/apps/case_importer/exceptions.py

```diff
@@ -96,6 +96,14 @@ class InvalidDate(CaseRowError):
     )
 
 
+class InvalidSelectValue(CaseRowError):
+    title = ugettext_noop('Unexpected muliple choice value')
```

Comment:
```suggestion
    title = ugettext_noop('Unexpected multiple choice value')
```

Reply:
Oops. I so badly need a spell checker in my code editor...